### PR TITLE
Requesting 16 cpu limit for smart-village-view

### PR DESCRIPTION
**Pull request by computate**
Hello Operate First team! Could I please increase the ResourceQuota of the smart-village-view project with a custom ResourceQuota with a 16 CPU limit? 

We are onboarding a new FIWARE Context Broker and other applications as part of our IoT Traffic Simulations. We also have large data imports and it seems to be limited by the database, search engine, cluster manager, and application having enough CPU limit to process the data.

### CODE REVIEW

File: cluster-scope/base/core/namespaces/smart-village-view/resourcequota.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: opf-ci-pipelines-custom
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/smart-village-view/resourcequota.yaml:4]: Nit, please change the name to match the namespace name.

```suggestion
  name: smart-village-view-custom
```

**Review by tumido — APPROVED** [review]
One small nit, otherwise lgtm :slightly_smiling_face:

**Review by tumido — CHANGES_REQUESTED** [review]
^

File: cluster-scope/base/core/namespaces/smart-village-view/resourcequota.yaml (hunk 1 above)
Comment (computate) [reply cluster-scope/base/core/namespaces/smart-village-view/resourcequota.yaml]: Great feedback @tumido , I have renamed my resourcequota.

**Review by tumido — APPROVED** [review]
:slightly_smiling_face: this looks good!

/lgtm
/approve

**Comment by computate** [comment]
Thank you so much for the new ResourceQuota!!!

**Pull request merged**